Ui/Component/Listing/Column/NotificationColumn.php

```diff
@@ -104,6 +104,14 @@ public function prepareDataSource(array $dataSource)
                     $item["pspreference"]
                 );
             }
+
+            if (!$item["done"]) {
+                $item["actions"] = sprintf(
+                    '<a href="%s">%s</a>',
+                    $orderUrl = $this->backendHelper->getUrl("adyen/notifications/reprocess", ["entity_id" => $item["entity_id"]]),
```

Comment:
Do we need this `$orderUrl` assignment in this expression? It would be better to just pass a value or move it to a separate statement.

Reply:
Hello Michael, thank you for your review. I sent a commit about your review.
https://github.com/Adyen/adyen-magento2/pull/1410/commits/a025cb2e702d44f2425ecadeaeaab1e11bad7bda